bootstrap_jenkins.sh

```diff
@@ -21,19 +21,15 @@ function cms-merge-commit()
 
 git cms-init --upstream-only
 
-echo "---> Merging recipes..."
-cms-merge-commit cms-egamma EGM_gain_v1 f2ae5ef247d2544bdccd0460586b468eb35631c5
-cms-merge-commit rafaellopesdesa RegressionCheckNegEnergy 3aafeff0371a1d1eb3db9d95ef50c1a66da25690
-cms-merge-commit cms-met METRecipe_8020_for80Xintegration 92f73cd3d16a9529585865a365de271e0535b68d
-
 echo "---> Running git cms-checkdeps"
 git cms-checkdeps -a
 
-echo "---> Adding KaMuCa"
-git clone -o upstream https://github.com/bachtis/analysis.git -b KaMuCa_V4 KaMuCa 
-pushd KaMuCa
-git checkout 2ad38daae37a41a9c07f482e95f2455e3eb915b0
-popd
-
 echo "---> Adding TreeWrapper"
 git clone -o upstream https://github.com/blinkseb/TreeWrapper.git cp3_llbb/TreeWrapper
+
+echo "---> Adding phase2 EGamma"
+git cms-merge-topic -u nsmith-:EgammaFromMultiCl_932v2
```

Comment:
This should go above `git cms-checkdeps -a` (you never know). We merge a specific commit to make sure things don't change when some adds a commit to a branch, but you probably don't need / want that level of paranoia in this case